ProjectToTest/Program.cs

```diff
@@ -1,205 +1,71 @@
-namespace HOMEWORK3_QUADRATIC_YPARKHOMENKO
+using ProjectToTest;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+internal class Program
 {
-    internal class Program
+    public static bool IsValidName(string name = "")
     {
-        static void Main()
-        {
-            // See https://aka.ms/new-console-template for more information
-            Console.WriteLine("Hello, World!");         // INITIAL HELLO.
-            Console.WriteLine("Hello from Daniil!");    // HELLO FROM STUDENT DANIIL.
-            Console.WriteLine("Hello from Nina!");      // HELLO FROM STUDENT NINA.
-            Console.WriteLine("Goodbye from Illia!");     // Goodbye FROM STUDENT ILLIA.
-            Console.WriteLine("For what its worth?");   // HELLO FROM STUDENT YAROSLAV.
-
-            Console.WriteLine($"\nTHIS PROGRAM SOLVING A QUADRATIC EQUATIONS OF THE FORM\n\nAx\u00B2 + Bx + C = 0,\n\nAND THE SECOND FORM\n\n(Px + Q)(Rx + S) = 0,\n\nA, P, B, Q, C, R, S — ARE COEFFITIENTS THE EACH OF THEM IS AN INTEGER NUMBER (MEANS NO FRACTIONAL PART) THAT CAN BE LOWER THAN 0, THE POSSIBLE RANGE OF INTEGER VALUES FOR A SINGLE VARIABLE ON ELECTRONIC CALCULATION MACHINE IS ({int.MinValue}, {int.MaxValue})\n\n");
-
-            byte CHECK_CONSOLE_AT_START = 0;
-            string SECOND_TYPE = "UNASSIGNED"; //ASSIGNMENT OF STRING VARIABLE THAT REQUIRED FOR OUR MECHANISM. TO READ IT IN "CATCH" PROGRAM.
+        Regex regex = new Regex(@"^[A-Za-z]+$");
+        return !string.IsNullOrWhiteSpace(name) && regex.IsMatch(name);
```

Comment:
IsMatch() a static stuff — no need the instance to call it — (https://learn.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.ismatch?view=net-8.0)